internal/drivers/ankabuild/util.go

```diff
@@ -0,0 +1,16 @@
+package ankabuild
+
+import "github.com/drone-runners/drone-runner-aws/internal/oshelp"
```

Comment:
combine util.go and client.go or move the tempdir function in with another file. it looks lonely

Reply:
done